app/controllers/application_controller/google_authentication_dependency.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class ApplicationController
```

Comment:
So I tried making a new dependency to add it to the `ApplicationController` but this didn't really work. The methods weren't added for some reason. Am I missing something?

Reply:
Ah! I ran into this issue too -- 

you need to add your file to the `depends_on:` line within `ApplicationController`

Reply:
Oh really? I knew I was missing something 😂